cmd/root.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright © 2025 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// fdo_server [-debug] print-owner-pubkey <type>
+// 	--db <path>
+// 	--db-pass <password>
+// fdo_server [-debug] import-voucher <path>
+// 	--db <path>
+// 	--db-pass <password>
+// fdo_server [-debug] resell <guid> <key-path>
+// 	--db <path>
+// 	--db-pass <password>
+// fdo_server [-debug] serve
+// 	[--insecure-tls]
+// 	[--server-cert=<path>]
+// 	[--server-key=<path>]
+// 	[--external-addr=<address>]
+// 	[--command-date]
+// 	[--command-wget=<url>]
+// 	[--command-upload=<file>]
+// 	[--command-upload-dir=<path>]
+// 	[--command-download=<file>]
+// 	[--reuse-credentials]
+// 	--db <path>
+// 	--db-pass <password>
+// 	<http-addr>
```

Comment:
@ben-krieger a draft of what this looks like

Reply:
Looks good. I would skip the `--command-XXX` flags. Even though that's existing functionality, we're going to want something more powerful.

I'm not exactly sure what that'll look like, so when we get there, please rely on me to help define and implement it. I think we'll likely want a whole subcommand for getting/setting complex per-device FSIM chains...

Reply:
agreed, I've initially added those --command-XXX just for the sake of supportability with the _old_ version - I think we might just keep them in this PR iteration so there's feature parity and then move away asap with a follow up (just for the sake of moving fast and share work)